modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/IgniteMethod.java

```diff
@@ -94,7 +95,10 @@ public enum IgniteMethod {
     GREATEST2(IgniteSqlFunctions.class, "greatest2", Object.class, Object.class),
 
     /** See {@link Objects#equals(Object, Object)}. */
-    IS_NOT_DISTINCT_FROM(Objects.class, "equals", Object.class, Object.class);
+    IS_NOT_DISTINCT_FROM(Objects.class, "equals", Object.class, Object.class),
+
+    /** See {@link UUID#randomUUID()}. */
+    RAND_UUID(UUID.class, "randomUUID");
```

Comment:
I think it would be better to name this function RANDOM_UUID, because abbreviations are mostly used in cases when a function is borrowed from an existing domain (such as math, finance, etc) and that name is widely adopted (so the name is defacto standard).

Reply:
I named it `RAND_UUID` because we have built-in `RAND` and `RAND_INTEGER`.
If you insist that it should be renamed, I will.

Reply:
btw, h2 has `RAND` and `RANDOM_UUID`